easybuild/easyconfigs/a/avogadroapp/avogadroapp-1.95.1-gompi-2021a.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'avogadroapp'
+version = '1.95.1'
+
+homepage = 'https://two.avogadro.cc/'
+description = """Avogadro is an advanced molecule editor and visualizer
+designed for cross-platform use in computational chemistry, molecular
+modeling, bioinformatics, materials science, and related areas. It
+offers flexible high quality rendering and a powerful plugin
+architecture.
+
+This is avogadroapp for Avogadro2.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
+
+source_urls = ['https://github.com/OpenChemistry/avogadroapp/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['d59694a6b32bb95317c66f4c806fe05d46e7e3ca4e594aa214bf9dd7e179b317']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Eigen', '3.3.9'),
+    ('Qt5', '5.15.2'),
+    ('avogadrolibs', '1.95.1'),
+]
+
+build_shared_libs = True
+
+configopts = '-DAvogadro_ENABLE_RPC=OFF -DBUILD_DOCUMENTATION=OFF '
+# configopts += '-DUSE_VTK=ON '
+
+sanity_check_paths = {
+    'files': ['bin/avogadro2'],
```

Comment:
Also add a sanity check command for `avogadro2`, something like `avogadro2 --help`?